File: HoWo7/PhoneCompany/Xiomi.py
from Phone.Smartphone import Smartphone;

class Xiomi(Smartphone):
    def __init__(self,model:str, sizeDisplay, akb: int, typeOfAntenna:str, osVerion:str, price:int):
        super().__init__(model, sizeDisplay, akb, price, ("Android " + osVerion) , typeOfAntenna)

    def __str__(self) ->str:
        return "Xiomi " + super().__str__()

File: HoWo5_6/PhoneCompany/Xiomi.java
package HoWo5_6.PhoneCompany;

import HoWo5_6.Phone.Smartphone;

public class Xiomi extends Smartphone{

    public Xiomi(String model, double sizeDisplay, int akb, String typeOfAntenna, String osVerion, int price) {
        super(model, sizeDisplay, akb, price, ("Android " + osVerion) , typeOfAntenna);
    }

    @Override
    public String toString() {
        return "Xiomi "+ super.toString();
    }
}
